src/outlet/index.ts

```diff
@@ -360,6 +360,8 @@ export class Outlet {
 					min_height: this.tokenConfig.whitelistDialogHeight,
 				})
 			})
+		} else {
```

Comment:
@nicktaras else is redundant since the if statement returns a promise, can you please remove it?

Reply:
Thanks for taking a look @micwallace could you run this code on your side too against the examples.

Where from testing my side, without the else the function will automatically return sending the tokens before permission is provided.

Reply:
Apologies my side on this. 

I'd forgotten we are now producing a second copy of the attestation inside the ticket issuer (main site) which I hadn't removed from storage, giving the impression that we could load tokens without permission granted.

I've resolved removing the else and re-tested and works perfectly.